src/components/CountdownCounter/CountdownCounter.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useEffect } from 'react';
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import { ThemeContext } from 'styled-components';
+import useInterval from 'src/hooks/useInterval';
+import { CountdownCounterProps } from './types';
+import 'react-circular-progressbar/dist/styles.css';
```

Comment:
I think this line will break nextjs, please try your component version here https://github.com/pangolindex/swap-widget-example/tree/master/next.js and see whether it works or not.

Reply:
It works well.

Reply:
fixed